cmake/app/boilerplate.cmake

```diff
@@ -71,6 +71,23 @@ set(__build_dir ${CMAKE_CURRENT_BINARY_DIR}/zephyr)
 
 set(PROJECT_BINARY_DIR ${__build_dir})
 
+if(${CMAKE_VERSION} VERSION_EQUAL 3.19.1)
```

Comment:
I used CMake `3.19.0` and had the same issue as described in https://github.com/zephyrproject-rtos/zephyr/issues/30232

Reply:
In your commit message, you mention issue 20232 at some point. I guess it is a typo and it should be #30232?

Reply:
> I used CMake 3.19.0 and had the same issue as described in #30232

3.19.0 now included in test and message.

> In your commit message, you mention issue 20232 at some point. I guess it is a typo and it should be #30232?

Thanks for spotting, fixed.